### ISSUE TITLE
Missing accessibility labels for some links and buttonson details page, making them difficult to use with a screen reader

### ISSUE THREAD

**Issue by pitermach** [haiwen/seahub#3813]
I'm using Seafile with a screen reader (NVDA on Windows, VoiceOver on Mac OS). A screen reader as the name suggests reads anything displayed on the screen. 

I just updated Seafile to a version with the redesigned UI, and most elements are read out correctly. However, there are a few notable exceptions which the screen reader just reads out as "link" or "button" which makes it impossible to figure out what they do.

Steps to reproduce:

- Launch a screen reader (On Windows I recommend NVDA because it's free and has the best web browser support. On a Mac, enable VoiceOver by pressing commmand-F5.
- Navigate to a file's details page.
- Explore the page. If using NVDA, use down arrow to read through the page, or on a Mac with VoiceOver, use CTRL-Option-Right and left arrow.

Expected: When you reach the group of controls that let you do things like download or share the file, the screen reader should identify them with a meaningful label, like Download, Share, Revision History and so on.
Actual: only the controls type is spoken.

Another effect of these labels not being there is that in addition to blind users of screen reading software, people using Speech Recognition software like Dragon on Windows or the upcoming voice command feature in Mac OS Catalina will not be able to click on these buttons either, as that software also relies on accessibility labels being present to identify controls.

**Comment by Neurrone** [comment]
Second this issue.

I'll try to see if I can find the location in the source where those particular html elements are constructed.

The fix is pretty trivial, just adding an `aria-label` attribute, set to the same value of the text that appears on hover.

**Comment by Neurrone** [comment]
Oh, has this been fixed?

**Comment by Neurrone** [comment]
At least as of 8.0.5, this has not been fixed.

**Comment by freeplant** [comment]
Related PR: https://github.com/haiwen/seahub/pull/4934
